rocks.kfs.FundraisingParticipantSummary/Properties/AssemblyInfo.cs

```diff
@@ -22,10 +22,10 @@
 // associated with an assembly.
 [assembly: AssemblyTitle( "rocks.kfs.FundraisingParticipantSummary" )]
 [assembly: AssemblyProduct( "rocks.kfs.FundraisingParticipantSummary" )]
-[assembly: AssemblyCopyright( "Copyright © Kingdom First Solutions 2020" )]
+[assembly: AssemblyCopyright( "Copyright © Kingdom First Solutions 2022" )]
 
 // Auto increment assembly versions
-[assembly: AssemblyVersion( "1.0.*" )]
+[assembly: AssemblyVersion( "1.1.*" )]
```

Comment:
Due to mishaps with version, I believe we are up to 1.4 now, double check in rock shop?
```suggestion
[assembly: AssemblyVersion( "1.4.*" )]
```

Reply:
Latest RockShop package show 1.2. We can go with 1.3 unless you have another reason for going to 1.4.

Reply:
1.3 is fine.